templates/wp-config.php.j2

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * The base configuration for WordPress
+ *
+ * The wp-config.php creation script uses this file during the
+ * installation. You don't have to use the web site, you can
+ * copy this file to "wp-config.php" and fill in the values.
+ *
+ * This file contains the following configurations:
+ *
+ * * MySQL settings
+ * * Secret keys
+ * * Database table prefix
+ * * ABSPATH
+ *
+ * @link https://codex.wordpress.org/Editing_wp-config.php
+ *
+ * @package WordPress
+ */
+
+// ** MySQL settings - You can get this info from your web host ** //
+/** The name of the database for WordPress */
+define('DB_NAME', '{{ wordpress_mysql_database_name }}');
+
+/** MySQL database username */
+define('DB_USER', '{{ wordpress_mysql_user }}');
+
+/** MySQL database password */
+define('DB_PASSWORD', '{{ wordpress_mysql_password | mandatory }}');

```

Comment:
looks like a mandatory var. Describe it in readme.
You may go ahead and describe all mandatory and useful vars in readme, even without explicit request for every single one of them...

Reply:
done